cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-kind: ReplicationController
+apiVersion: apps/v1beta1
+kind: StatefulSet
```

Comment:
I tested Deployment, and in general, it did as well as StatefulSet. 
I left StatefulSet, because during my tests I found it was up a little bit faster. The difference was just a couple of seconds, though and I even haven't try to reproduce those results :) so I'm open to switching it to Deployment.

And why would Deployment be a better choice?